client/components/TestRenderer/CommandResults/index.jsx

```diff
@@ -37,21 +36,14 @@ const CommandResults = ({
   });
   const tooltipID = useMemo(() => `untestable-tooltip-${++tooltipCount}`, []);
 
-  const hasUnresolvedAssertions = assertions.some(
-    assertion => assertion.passed === null
-  );
-
-  const hasConflictingOutput = isRerunReport && hasUnresolvedAssertions;
-
-  const errorMessage = hasConflictingOutput ? 'Conflicting Output' : null;
-
-  const shouldShowHistoricalOutput =
-    hasConflictingOutput && historicalOutput !== null;
+  const matchType = match?.type;
+  const isNoMatch = matchType === 'NONE';
+  const errorMessage = isRerunReport && isNoMatch ? 'Conflicting Output' : null;
```

Comment:
@stalgiag are we also rendering and counting the rare "Incomplete" match scenarios as matches? If so, is there any UI indicating that they're "incomplete" matches?